### PR TITLE
Implemented Select/Deselect group

### LINKED ISSUE #39: Implement Select/Deselect group
The menu items are already there, but they do nothing.

### PR DESCRIPTION
**Pull request by ocerman**
This PR implements Select/Deselect group. User can define group by fnmatch filename pattern, like "*.zip"

### CODE REVIEW

File: src/GenesisSelectGroupWindow.h [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ * Distributed under the terms of the MIT license.
+ *
+ *	2019-2020 Ondrej Čerman
+ */
+#ifndef GENESISSELECTGROUPWINDOW_H
+#define GENESISSELECTGROUPWINDOW_H
+
+#include "GenesisCustomListView.h"
+
+#include <Window.h>
```
Comment (humdingerb) [inline src/GenesisSelectGroupWindow.h]: Alpha-sort headers

**Review by humdingerb — CHANGES_REQUESTED** [review]
Works nicely!
There are quite a few stray whitespaces in there. Koder can show them very well (activate in its settings).

**Comment by ocerman** [comment]
@humdingerb Didn't know that Koder have this feature, thanks.

I have removed trailing whitespaces in newly created files, I will try to clean all other project files later.

**Comment by humdingerb** [comment]
Thanks!

**Pull request merged**